gradio/blocks.py

```diff
@@ -603,6 +624,9 @@ def iterate_over_children(children_list):
                         iterate_over_children(children)
 
         with Blocks(theme=config["theme"], css=config["theme"]) as blocks:
+            # ID 0 should be the root Blocks component
+            original_mapping[0] = Context.root_block or blocks
```

Comment:
Fixing a different bug that was preventing `gr.Interface.load()` from working if a dataset had example caching on